conftest.py

```diff
@@ -127,6 +148,84 @@ def pytest_keyboard_interrupt(excinfo):
         exp.stop()
 
 
+# The callback for when the client receives a CONNACK response from the server.
```

Comment:
Can we move this in a doctring instead of a comment ? (same for on_message below)

Reply:
yes, I can